p2p/src/lib.rs

```diff
@@ -63,59 +61,39 @@ where
         config: Arc<ChainConfig>,
     ) -> error::Result<Self> {
         let (conn, flood) = T::start(addr, &[], &[]).await?;
-        let swarm_config = Arc::clone(&config);
         let (tx_swarm, rx_swarm) = mpsc::channel(16);
+        let (tx_sync, rx_sync) = mpsc::channel(16);
```

Comment:
There isn't. It's on my todo list to make sure our blockchain doesn't choke when those channels are full and that they drop messages appropriately/otherwise handle the situation gracefully but I just haven't gotten around to it yet. Setting a low number like 16 doesn't give a false sense of security so I actually remember to write those tests when I have the time as the number reminds me every time I see it.